tests/plugins/test_resolve_remote_source.py

```diff
@@ -844,6 +850,7 @@ def test_allow_multiple_remote_sources(workflow, allow_multiple_remote_sources):
                     "gomod/cachito.env": first_cachito_env,
                     "gomod/app/README.txt": "Content of remote-source-gomod.tar.gz",
                     "gomod/app/some-config.txt": "gomod requests don't actually have configs",
+                    "gomod/app/.bundle/config": "RubyGems config contents",
```

Comment:
@chmeliik Not sure if I completely understood, but I quite changed it, just here it doesn't make completely sense, because it seems to me that this test is using it in a way that it's gonna be part of gomod package.

Reply:
Hmm, true. It doesn't really matter, the main point is that the parent directory doesn't exist, but if you want to make it less confusing you could change the path/content to something less ruby-specific

And I think we don't need the `test_generate_cachito_config` test anymore, since config files are tested as part of the larger tests

Reply:
Yeah, I made it less ruby specific, it makes much sense now and double-checked that `test_generate_cachito_config` can be removed (tests failed when I uncommented my changes).